src/main/scala/com/databricks/labs/overwatch/pipeline/GoldTransforms.scala

```diff
@@ -493,6 +493,82 @@ trait GoldTransforms extends SparkSessionWrapper {
 
   }
 
+
+  protected def buildNotebookCommands(
+                                   notebook: PipelineTable,
+                                   clsfIncrementalDF : DataFrame,
+                                 )(auditIncrementalDF: DataFrame): DataFrame = {
+
+
+    val notebookLookupTSDF = notebook.asDF
+      .select("organization_id", "notebook_id", "notebook_path", "notebook_name", "unixTimeMS","date")
+      .withColumnRenamed("notebook_id", "notebookId")
+      .filter('notebookId.isNotNull)
+      .distinct
+      .toTSDF("unixTimeMS", "organization_id","date","notebookId")
+
+    val clsfLookupTSDF = clsfIncrementalDF
+      .select(
+        "organization_id", "state_start_date", "unixTimeMS_state_start", "cluster_id", "cluster_name",
+        "custom_tags", "node_type_id", "current_num_workers", "uptime_in_state_H", "total_cost")
+      .distinct
+      .withColumnRenamed("unixTimeMS_state_start", "unixTimeMS")
+      .withColumnRenamed("cluster_id", "clusterId")
+      .withColumnRenamed("current_num_workers", "node_count")
+      .withColumn("totalCostPMS", col("total_cost") / col("uptime_in_state_H")/lit(3600))
```

Comment:
totalCostPMS ?? What is this? I thought this was totalCost per millisecond, is that right? If so why are you dividing hours by 3600? Furthermore, if it's millis it should be Hour * 3600 * 1000, right?

If the `executionTime` is in seconds, then you want this in seconds at which point hours * 3600 would be to the seconds but then you should change the name to be total_dbu_cost_ps or something similar to notate that it's in seconds.